Ginger/Ginger/Actions/ApiActionsConversion/ApiActionsConversionWizard.cs

```diff
@@ -0,0 +1,249 @@
+                if(isPresent)
+                {
+                    break;
+                }
+            }
+            return isPresent;
+        }
+
+        /// <summary>
+        /// This is finish method which does the finish the wizard functionality
+        /// </summary>
+        public override void Finish()
+        {
+            base.mWizardWindow.Close();
+        }
+
+        /// <summary>
+        /// This method is used to Stop the conversion process in between conversion process
+        /// </summary>
+        public void StopConversion()
+        {
+            mConversionUtils.StopConversion();
+        }
+
+        /// <summary>
+        /// This method is used to convert the action in case of Continue & Re-Convert
+        /// </summary>
+        /// <param name="lst"></param>
+        /// <param name="isReConvert"></param>
+        public async void ProcessConversion(ObservableList<BusinessFlowToConvert> lst, bool isReConvert)
+        {
+            ProcessStarted();
+            try
+            {
+                ObservableList<BusinessFlowToConvert> flowsToConvert = new ObservableList<BusinessFlowToConvert>();
+                if (isReConvert)
+                {
+                    ObservableList<BusinessFlowToConvert> selectedLst = new ObservableList<BusinessFlowToConvert>();
+                    foreach (var bf in lst)
+                    {
+                        if (bf.IsSelected)
+                        {
+                            bf.BusinessFlow.RestoreFromBackup(true);
+                            bf.ConversionStatus = eConversionStatus.Pending;
+                            bf.SaveStatus = eConversionSaveStatus.Pending;
+                            flowsToConvert.Add(bf);
+                        }
+                    }
+                }
+                else
+                {
+                    flowsToConvert = ListOfBusinessFlow;
+                }
+
+                if (flowsToConvert.Count > 0)
+                {
+                    await Task.Run(() => mConversionUtils.ConvertToApiActionsFromBusinessFlows(flowsToConvert, ParameterizeRequestBody, PullValidations));
+                }
+                mReportPage.SetButtonsVisibility(true);
+            }
+            catch (Exception ex)
+            {
+                Reporter.ToLog(eLogLevel.ERROR, "Error occurred while trying to convert", ex);
+            }
+            finally
+            {
+                ProcessEnded();
+            }
+        }
+
+        /// <summary>
+        /// This method is used to convert the actions
+        /// </summary>
+        /// <param name="lst"></param>
+        public async void BusinessFlowsActionsConversion(ObservableList<BusinessFlowToConvert> lst)
+        {
+            try
+            {
+                ProcessStarted();
+
+                await Task.Run(() => mConversionUtils.ConvertToApiActionsFromBusinessFlows(lst, ParameterizeRequestBody, PullValidations));
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add '.ConfigureAwait(false)' to this call to allow execution to continue in any thread.](https://app.codacy.com/app/Ginger-Automation/Ginger/pullRequest?prid=4464228)